app_dart/test/request_handlers/github_webhook_test.dart

```diff
@@ -1048,36 +1071,195 @@ void main() {
+},
+{
+  "requests":[
+    {
+      "searchBuilds":{
+        "predicate":{
+          "builder":{
+            "project":"flutter",
+            "bucket":"try"
+          },
+          "createdBy":"test@test",
+          "tags":[
+            {
+              "key":"buildset",
+              "value":"pr/git/583"
+            },
+            {
+              "key":"github_link",
+              "value":"https://github.com/flutter/cocoon/pull/583"
+            },
+            {
+              "key":"user_agent",
+              "value":"flutter-cocoon"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "searchBuilds":{
+        "predicate":{
+          "builder":{
+            "project":"flutter",
+            "bucket":"try"
+          },
+          "tags":[
+            {
+              "key":"buildset",
+              "value":"pr/git/583"
+            },
+            {
+              "key":"user_agent",
+              "value":"recipe"
+            }
+          ]
+        }
+      }
+    }
+  ]
+},
+{
+  "requests":[
+    {
+      "scheduleBuild":{
+        "builder":{
+          "project":"flutter",
+          "bucket":"try",
+          "builder":"Cocoon"
+        },
+        "properties":{
+          "git_url":"https://github.com/flutter/cocoon",
+          "git_ref":"refs/pull/583/head"
+        },
+        "tags":[
+          {
+            "key":"buildset",
+            "value":"pr/git/583"
+          },
+          {
+            "key":"buildset",
+            "value":"sha/git/the_head_sha"
+          },
+          {
+            "key":"user_agent",
+            "value":"flutter-cocoon"
+          },
+          {
+            "key":"github_link",
+            "value":"https://github.com/flutter/cocoon/pull/583"
+          }
+        ],
+        "notify":{
+          "pubsubTopic":"projects/flutter-dashboard/topics/luci-builds",
+          "userData":"eyJyZXRyaWVzIjowfQ=="
+        }
+      }
+    }
+  ]
+},
+{
+  "requests":[
+    {
+      "searchBuilds":{
+        "predicate":{
+          "builder":{
+            "project":"flutter",
+            "bucket":"try"
+          },
+          "createdBy":"test@test",
+          "tags":[
+            {
+              "key":"buildset",
+              "value":"pr/git/583"
+            },
+            {
+              "key":"github_link",
+              "value":"https://github.com/flutter/cocoon/pull/583"
+            },
+            {
+              "key":"user_agent",
+              "value":"flutter-cocoon"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "searchBuilds":{
+        "predicate":{
+          "builder":{
+            "project":"flutter",
+            "bucket":"try"
+          },
+          "tags":[
+            {
+              "key":"buildset",
+              "value":"pr/git/583"
+            },
+            {
+              "key":"user_agent",
+              "value":"recipe"
+            }
+          ]
+        }
+      }
+    }
+  ]
+}]''';
```

Comment:
Nit: it's still hard to understand what's going on in this big json dump.

Reply:
Unfortunately there is no much we can do here. We can use counts only but we will lose coverage.